spec/lib/finder_api_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "spec_helper"
 
 describe FinderApi do
-  context "when merging and de-duplicating sorting by" do
+  describe "when merging and de-duplicating sorting by" do
```

Comment:
I don't think there's much difference between describe and context. Could we just use one for consistency? Perhaps squash up those changes to when the test was first updated, to make it easier to see how the tests have changed because of the addition of "relevance" to the results.